web/css/source/_theme.less

```diff
@@ -324,6 +370,20 @@ h1.site-intro{
   }
 }
 
+// hackity hack hack - Maganto style! - overriding Luma one ID at a time as I couldn't get to work any other way
+a#ui-id-1.level-top.ui-corner-all,
+a#ui-id-2.level-top.ui-corner-all,
+a#ui-id-3.level-top.ui-corner-all,
+a#ui-id-4.level-top.ui-corner-all,
+a#ui-id-5.level-top.ui-corner-all,
+a#ui-id-6.level-top.ui-corner-all,
+a#ui-id-7.level-top.ui-corner-all,
+a#ui-id-8.level-top.ui-corner-all
+a#ui-id-9.level-top.ui-corner-all,
+a#ui-id-10.level-top.ui-corner-all{
+  color: white;
```

Comment:
Just looking at this, the `#ui-id-X` ID format comes from jQuery, and so should probably not be relied on.  The `.ui-corner-all` is also quite generic, and so probably only adds some specificity (but only as much as a class, which is nothing compared to the ID selector).

Looking at yume, `web/css/magento/menu.less` (which we had overridden but you are removing in this PR) line 26 has `color: #fff !important;`.  So maybe you were overriding that, so this was needed at the time, but now is not required since taking yume out of the mix.  Might be worth a try.

Failing that, I'd be more comfortable with a parent ID selector than relying on the jQuery generated ones.  Maybe find a parent element with an ID, and use that: `#id-of-parent a.level-top {...}` (or with the `.ui-corner-all`).